autopts/ptscontrol.py

```diff
@@ -513,6 +513,8 @@ def restart_pts(self, args=None):
 
         log("restart_pts")
         exception = 0
+        dongle_init_retry = args.dongle_init_retry if args else 5
+
```

Comment:
Suggest to put this into the exception scope

Reply:
Changed to use `getattr` instead, as it handles cases when args is None and `dongle_init_retry` does not exist in args.